.github/workflows/nightly_fuzzing.yml

```diff
@@ -8,39 +8,19 @@ on:
 permissions: read-all
 
 jobs:
-  tests-unit:
+  unit-tests:
     runs-on: ubuntu-latest-64-cores
     timeout-minutes: 1440
     env:
       HYPOTHESIS_PROFILE: nightly
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Set up Rust toolchain
-        run: rustup show
-
-      - uses: astral-sh/setup-uv@v2
-        with:
-          enable-cache: true
-          cache-dependency-glob: uv.lock
-      - uses: actions/setup-python@v5
-        with:
-          python-version-file: .python-version
-      - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1
-        with:
-          version: nightly
-      - name: Install forge deps
-        run: forge install
-      - name: Build solidity contracts
-        run: forge build
+      - uses: ./.github/actions/setup-environment
       - name: Run unit tests
         id: unit-tests
         continue-on-error: true
         run: |
-          uv run --reinstall pytest -n logical --durations=0 -v -s --log-cli-level=DEBUG --no-skip-cached-tests --ignore-glob=cairo/tests/ef_tests/
-
+          uv run pytest -n logical --durations=0 -v -s --log-cli-level=DEBUG --no-skip-cached-tests --ignore-glob=cairo/tests/ef_tests/
```

Comment:
`--ignore-glob=cairo/tests/ef_tests/` is redundant with pyproject.toml

Reply:
see `norecursedirs = ".* cairo/tests/ef_tests/fixtures"`

Reply:
no this is different, one is ignoring all ef-tests, the other is just the fixtures data (which contain some python code)